week-3/alarmclock/alarmclock.test.js

```diff
@@ -1,3 +1,4 @@
+
 /* ======= TESTS - DO NOT MODIFY ===== 
```

Comment:
**!!! IMPORTANT !!!** In line 1 it says 'TESTS - DO NOT MODIFY.' That means it's important **not to change** those tests. If you see errors from those tests, it's like a signal that something might be wrong with your code. Instead of deleting the tests, try to figure out a solution that makes them work correctly

Reply:
Thanks Musa